### PR TITLE
Modified docs for SaaS 

### PR DESCRIPTION
**Pull request by josephrosenberg**
to include references to the SaaS, fixes to the CLI lan…guage, and added a new SaaS specific tutorial

### CODE REVIEW

**Comment by josephrosenberg** [comment]
I renamed tutorial to tutorial-local, and changed tutorial to be the SaaS version so all old links to tutorial didn't need to be updated.

File: docs/getting-started/saas_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+CREATE DATABASE tutorial;
+```
+
+Exit out of the `psql` application with the following command:
+
+```sh
+\q
+```
+
+## Create an Account in the Mycelial App
```
Comment (tbaums) [inline docs/getting-started/saas_tutorial.md]: can we pick something other than "app"? like can we call it Mycelial Cloud or or maybe we just say "Create a Mycelial Account"

File: docs/getting-started/saas_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+```sh
+mycelial destroy
+```
+
+Run the following commands to stop and remove the Postgres database:
+
+```sh
+docker stop postgres-db
+docker rm postgres-db
+```
```
Comment (tbaums) [inline docs/getting-started/saas_tutorial.md]: this is great

**Review by tbaums — APPROVED** [review]
looks great!

**Pull request merged**